ksqldb-cli/src/main/java/io/confluent/ksql/cli/console/KsqlLineParser.java

```diff
@@ -47,6 +47,10 @@ public ParsedLine parse(final String line, final int cursor, final ParseContext
       return parsed;
     }
 
+    if (UnclosedQuoteChecker.isUnclosedQuote(line)) {
```

Comment:
I'm not thought too much about this, as its been a while since I've been in this code... but are _you_ happy with being passed `line` and not `parsed.line()`, i.e. do you understand the difference? (I certainly can't remember of the top of my head! )